tutorials/visualisation/graphics/canvas_divide_example.C

```diff
@@ -0,0 +1,118 @@
+// variants: 0  - custom values
+//           1  - default values
+//           2+ - old default values
```

Comment:
Maybe specify that old refers to anything before 6.40

Reply:
No chance of having this in 6.38.0?